src/miranda/convert/ghcn.py

```diff
@@ -189,7 +189,7 @@ def create_ghcn_xarray(
                         ds = ds.assign_coords(
                             {
                                 cc: xr.DataArray(
-                                    df_stat[cc].values[0], coords=ds.station.coords
+                                    [df_stat[cc].values[0]], coords=ds.station.coords
```

Comment:
Would this be more accurate?
```suggestion
                                    np.array(df_stat[cc].values[0]), coords=ds.station.coords
```

Reply:
Weird, but OK.